src/ansys/units/dimensions.py

```diff
@@ -26,7 +26,7 @@ def __init__(
         if units and dimensions:
             raise DimensionsError.EXCESSIVE_PARAMETERS()
 
-        self._units = ansunits.Units()
+        self._units = ansunits.Units
```

Comment:
I think we no longer need to treat these as if they are objects. Instead we can just call the function via the module.

Reply:
Ok I can make those changes. Would you be able to give me an example?

Reply:
Just remove the member variable `self._units` and you will be forced into the right thing.